evm_loader/tests/test_execute_trx_from_instruction.py

```diff
@@ -0,0 +1,319 @@
+            logs += " "
+        assert text in logs, f"Transaction logs don't contain '{text}'. Logs: {logs}"
+
+    def test_simple_transfer_transaction(self, operator_keypair, treasury_pool, sender_with_tokens, second_user,
+                                         evm_loader):
+        amount = 10
+        sender_balance_before = get_neon_balance(solana_client, sender_with_tokens.solana_account_address)
+        recipient_balance_before = get_neon_balance(solana_client, second_user.solana_account_address)
+
+        signed_tx = make_eth_transaction(second_user.eth_address, None, sender_with_tokens.solana_account,
+                                         sender_with_tokens.solana_account_address, amount)
+        resp = execute_trx_from_instruction(operator_keypair, evm_loader, treasury_pool.account, treasury_pool.buffer,
+                                            signed_tx,
+                                            [sender_with_tokens.solana_account_address,
+                                             second_user.solana_account_address],
+                                            operator_keypair)
+        sender_balance_after = get_neon_balance(solana_client, sender_with_tokens.solana_account_address)
+        recipient_balance_after = get_neon_balance(solana_client, second_user.solana_account_address)
+        assert sender_balance_before - amount == sender_balance_after
+        assert recipient_balance_before + amount == recipient_balance_after
+        self.check_transaction_logs_have_text(resp.value, "ExitSucceed")
+
+    def test_transfer_transaction_with_non_existing_recipient(self, operator_keypair, treasury_pool, sender_with_tokens,
+                                                              evm_loader):
+        # recipient account should be created
+        recipient = Keypair.generate()
+
+        recipient_ether = eth_keys.PrivateKey(recipient.secret_key[:32]).public_key.to_canonical_address()
+        recipient_solana_address, _ = evm_loader.ether2program(recipient_ether)
+        amount = 10
+        signed_tx = make_eth_transaction(recipient_ether, None, sender_with_tokens.solana_account,
+                                         sender_with_tokens.solana_account_address, amount)
+        resp = execute_trx_from_instruction(operator_keypair, evm_loader, treasury_pool.account, treasury_pool.buffer,
+                                            signed_tx,
+                                            [sender_with_tokens.solana_account_address,
+                                             PublicKey(recipient_solana_address)],
+                                            operator_keypair)
+        recipient_balance_after = get_neon_balance(solana_client, PublicKey(recipient_solana_address))
+        self.check_transaction_logs_have_text(resp.value, "ExitSucceed")
+
+        assert recipient_balance_after == amount
+
+    def test_deploy_contract(self, operator_keypair, evm_loader, treasury_pool, sender_with_tokens):
```

Comment:
test has positive name, but it wait for exception, it's negative case? What it check, why deploy not allowed?

Reply:
Because it should be small transaction
There are checks, if it deploy type,  neon evm send this error https://github.com/neonlabsorg/neon-evm/blob/develop/evm_loader/program/src/instruction/transaction_execute_from_instruction.rs#L95

I will change test name